lib/recurly/redemption_list.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class Recurly_CouponRedemptionList extends Recurly_Pager
+{
+  public static function get($params = null, $client = null)
+  {
+    $list = new Recurly_CouponRedemptionList(Recurly_Client::PATH_COUPON_REDEMPTIONS, $client);
+    $list->_loadFrom(Recurly_Client::PATH_COUPON_REDEMPTIONS, $params);
+    return $list;
+  }
```

Comment:
...and this, since they're not being used. Creating this class and plugging in `'redemptions' => 'Recurly_CouponRedemptionList',` was the important part of the fix.